contracts/feirari/RewardsDistributorAdmin.sol

```diff
@@ -0,0 +1,100 @@
+pragma solidity ^0.8.0;
+
+import "../refs/CoreRef.sol";
+import "./IRewardsDistributorAdmin.sol";
+
+contract RewardsDistributorAdmin is IRewardsDistributorAdmin, CoreRef {
+    /// @notice rewards distributor contract address
+    IRewardsDistributorAdmin public rewardsDistributorContract;
+
+    /// @param coreAddress address of core contract
+    /// @param _rewardsDistributorContract address of the admin rewards distributor contract
```

Comment:
Constructor param is a contract, but you're calling it an address here.

Reply:
Fixed